### DISCUSSION TITLE
Support zim wiki for macOS Ventura

### DISCUSSION THREAD

**Discussion by sitnjluk** [zim-desktop-wiki/zim-desktop-wiki#2279] [Q&A] [answered]
Hello,
I've just installed a new version of macOS named Ventura. Unfortunately zim wiki stopped to work and I found out from website that It supports macOS High Sierra up to macOS Monterey. Are there plans it supported macOS Ventura in the future as well? Thank you.

**Comment by dehesselle** [comment, marked as answer]
Yes, the plans exist and came to a halt due to upstream bugs (see https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/2219). They have been resolved (although not with 100% confidence) about 2-3 weeks ago and now it's just a matter of "once I have the time to do the necessary work" to produce the next release.

**Reply by dehesselle** [reply]
Nope, just tested a new build and the situation has not improved for Zim.

**Reply by dehesselle** [reply]
macOS Ventura is now supported.
